examples/multiband_pwgan/conf/multiband_pwgan.v1ft.yaml

```diff
@@ -0,0 +1,99 @@
+
```

Comment:
write a note that this config is tune on ur dataset so user need to adjust by theirself in their case. The differences are `discriminator_train_start_steps` = 0 and learning rate = ... :D